### DISCUSSION TITLE
Any way to make photon return street addresses w house numbers instead of points of interest or just roads?

### DISCUSSION THREAD

**Discussion by schrodingerscatz** [komoot/photon#672] [Q&A]
Finding an issue on my instance that the return for a reverse geocoding query tends to be streets or random geographical features instead of addresses with house numbers despite being on that position. Is there any way to try to force it to return addresses with house numbers and ignore other results?

**Comment by lonvia** [comment]
Not yet. See discussion about implementing something like it in https://github.com/komoot/photon/pull/667#issuecomment-1110689146
